packages/dev/core/src/FlowGraph/flowGraphInteger.ts

```diff
@@ -0,0 +1,93 @@
+import { RegisterClass } from "../Misc/typeStore";
+
+/**
+ * @experimental
+ * Class that represents an integer value.
+ */
+export class FlowGraphInteger {
+    /**
+     * The value of the integer. Its type
+     * is a javascript number. Shouldn't be
+     * directly modified - it is populated by
+     * the constructor.
+     */
+    readonly value: number;
+
+    constructor(value: number) {
+        this.value = this.toInt(value);
+    }
+
+    /**
+     * Converts a float to an integer.
+     * @param n the float to convert
+     * @returns the result of n | 0 - converting it to a int
+     */
+    public toInt(n: number): number {

```

Comment:
Just realized - this should either be static, or take no parameters and return the value to int. Or be kept private?

Reply:
I can make it private!